### PR TITLE
Reserved words and split strings.

### PR DESCRIPTION
**Pull request by pauleveritt**

### CODE REVIEW

File: pep.rst [hunk 1]
```diff
@@ -250,6 +250,11 @@ means ``app.f`` would be allowed, but discouraged:
 
     app.f'Hello {name}'
 
+Python `restricts certain keywords <https://docs.python.org/3/reference/lexical_analysis.html#keywords>`_ from being
+used as identifiers. This restriction also applies to tag names, including in dotted names. Usage of keywords should
+trigger a helpful error, as done in recent CPython releases.  The simplest solution would sure be to move reserved word
+recognition into the lexer.
```
Comment (lysnikolaou) [inline pep.rst:256]: Let's not mention implementation details here.

```suggestion
trigger a helpful error, as done in recent CPython releases.
```

**Review by lysnikolaou — APPROVED** [review]
LGTM!

File: pep.rst [hunk 2]
```diff
@@ -484,6 +488,23 @@ the following, at the cost of losing some type specificity:
     def mytag(*args: str | tuple) -> Any:
         ...
 
+A user might write a tag string as a split string:
```
Comment (jimbaker) [inline pep.rst:491]: Let's revisit this part of the specification. The current problem here I mention is that
```
tag"\N{{GRINNING FACE}}"
```
is called with
```
tag('\\N{', 'GRINNING FACE}')
```
so one raw string is resulting into two str args. We want the specification to disallow this split. and therefore it's an implementation bug, not just one possible way to implement the spec.

So something like the following should work:
> Any string parts of the tag string can only be split by an interpolation.

And then we can show this example.

File: pep.rst [hunk 3]
```diff
@@ -484,6 +488,23 @@ the following, at the cost of losing some type specificity:
...
+
+.. code-block:: python
+
+    def tag(*args):
+        return args
+
+    tag"\N{{GRINNING FACE}}"
+
+Tag strings will represent this as exactly one ``Decoded`` argument. In this case, ``Decoded.raw`` would be
```
Comment (jimbaker) [inline pep.rst:500]: +1 for how this part is described.

**Pull request merged**